ultradns_test/client_test.go

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright 2012-2013 NeuStar, Inc. All rights reserved. NeuStar, the Neustar logo and related names and logos are
+ * registered trademarks, service marks or tradenames of NeuStar, Inc. All other product names, company names, marks,
+ * logos and symbols may be trademarks of their respective owners.
+ */
+package ultradns_test
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/ultradns/ultradns-go-sdk/ultradns"
+)
+
+var (
+	testUsername          = os.Getenv("ULTRADNS_UNIT_TEST_USERNAME")
+	testPassword          = os.Getenv("ULTRADNS_UNIT_TEST_PASSWORD")
+	testHost              = os.Getenv("ULTRADNS_UNIT_TEST_HOST_URL")
+	testVersion           = os.Getenv("ULTRADNS_UNIT_TEST_API_VERSION")
+	testUserAgent         = os.Getenv("ULTRADNS_UNIT_TEST_USER_AGENT")
+	testZoneName          = os.Getenv("ULTRADNS_UNIT_TEST_ZONE_NAME")
```

Comment:
I think we should be using random zone names for our tests. Please convert this env variable to a local variable and add some random component so that every time some random data is created.